projects/js-packages/components/components/product-price/index.tsx

```diff
@@ -46,7 +47,8 @@ const ProductPrice: React.FC< ProductPriceProps > = ( {
 				) }
 				{ promoLabel && <Text className={ styles[ 'promo-label' ] }>{ promoLabel }</Text> }
 			</div>
-			{ <Text className={ styles.leyend }>{ leyend }</Text> }
+			{ ! children && <Text className={ styles.leyend }>{ leyend }</Text> }
+			{ children }
```

Comment:
Something to tackle in a follow-up PR:

```suggestion
			{ children ? children : <Text className={ styles.leyend }>{ leyend }</Text> }
```

Reply:
Feedback addressed in 4b8c663